### ISSUE TITLE
Breakpoint warning: The source code is different from the original version.

### ISSUE THREAD

**Issue by theSdev** [OmniSharp/omnisharp-vscode#2345]
Hi, I have updated my project to target .net core 2.1 yesterday using [this guide](https://blogs.msdn.microsoft.com/webdev/2018/05/07/asp-net-core-2-1-0-rc1-now-available/) and since then breakpoints in some files (which I later found out were changed or new files) were grayed out and didn't hit giving this message on insertion:

> Breakpoint warning: The source code is different from the original version. To allow this breakpoint to be hit: Add '"requireExactSource": false' to launch.json and restart debugging.

And when I did add that flag, I got this:

> Breakpoint warning: No executable code of the debugger’s target code type is associated with this line.

So it still didn't work. After banging my head against the wall and cursing the RC for hours I finally found the problem: In launch.json, the program flag was set to:

> "program": "${workspaceFolder}/bin/Debug/netcoreapp2.0/ProjectName.dll"

So basically I had to change the path to netcoreapp2.1.
I thought dropping this here may help someone, as ASP.NET Core 2.1 RTM release is close.
Thanks.

**Comment by duyn9uyen** [comment]
I had to do the same when I upgraded from .NET 5.0 to .NET 6.0. I had to change. Thanks!
> "program": "${workspaceFolder}/bin/Debug/net6.0/ProjectName.dll",
